api/src/router/api/v1/townies/list.ts

```diff
@@ -0,0 +1,27 @@
+"use strict";
```

Comment:
probably can remove once the tests are all working.  This was just added to do some manual testing